packages/graphql-tool-utilities/augmentations.ts

```diff
@@ -3,6 +3,7 @@ import {GraphQLProjectConfig} from 'graphql-config/lib/GraphQLProjectConfig';
 declare module 'graphql-config/lib/GraphQLProjectConfig' {
   interface GraphQLProjectConfig {
     resolveProjectName(defaultName?: string): string;
+    resolveSchemaPath(): string;
```

Comment:
Any reason not to do this as a getter?

Reply:
no particular reason, probably just to be consistent with `resolveProjectName`.